docs/source/getting-started/contributing.md

```diff
@@ -75,6 +249,12 @@ user@ubuntu:~/caliper$ node ./packages/caliper-cli/caliper.js launch manager \
     --caliper-networkconfig networks/fabric/test-network.yaml
 ```
 
+## Creating New SUT Connectors
+
+Connectors are relatively heavy components in Caliper. Before you attempt to create a new connector for a new SUT type, consult with the Caliper maintainers (on Discord, or in the form of a feature request).
+
+> More importantly, make sure that you are overly familiar with the documentation page about [implementing new connectors](https://hyperledger.github.io/caliper/v0.6.0/connectors/writing-connectors/).

```

Comment:
same here about using relative links rather than absolute links. 
I've also just realised that absolute links are everywhere in the docs so if it is possible to use relative links then we would need to fix those as well.

Reply:
Relative links means without the version?